packages/cxl-lumo-styles/scss/typography.scss

```diff
@@ -64,3 +66,25 @@ h6 {
 p {
   font-family: var(--cxl-lumo-font-secondary);
 }
+
+blockquote {
+  --lumo-secondary-text-color: var(--lumo-tint);
+  position: relative;
+  box-sizing: border-box;
+  padding: 1em var(--lumo-space-xl);
```

Comment:
Use design system space units instead of hardcoded values, outside very special situations.

https://cdn.vaadin.com/vaadin-lumo-styles/1.6.0/demo/sizing-and-spacing.html#spacing

`--lumo-space-wide-s` or larger probably works.